### PR TITLE
Don't query Tiny Machine for whitespace-only terms (closes #120).

### LINKED ISSUE #120: We should not query Tiny Machine for whitespace-only strings.
It's pretty wasteful.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @6a68

### CODE REVIEW

File: src/adapters/recommended_adapter.js [hunk 1]
```diff
@@ -7,7 +7,17 @@ class RecommendedAdapter {
   constructor () {
     this.results = new RecommendedResults();
     webChannel.on('printable-key', debounce((data) => {
-      this.search(data.query);
+
+      // Trim whitespace before performing search.
+      let query = data.query.replace(/^\s+/, '').replace(/\s+$/, '');
```
Comment (pdehaan) [inline src/adapters/recommended_adapter.js]: Why not just use `.trim()`?

Comment (chuckharmston) [inline src/adapters/recommended_adapter.js]: I originally had a more complex regex, but ended up stripping it down to this, which is functionally close to `.trim()`. Good catch.

**Comment by jaredhirsch** [comment]
Looks good! Swap out that regex for `trim` and feel free to merge. I'll open a parallel bug not to send anything over if it's just whitespace

**Comment by chuckharmston** [comment]
Thanks!

**Pull request merged**